views/eApostilles/openEApp.ejs

```diff
@@ -148,4 +148,5 @@
 
         <p>Your payment will appear on your bank statement as FCDO Legalisation Service.</p>
 
-    </section>
\ No newline at end of file
+        <a href="/download-receipt/<%= applicationId %>" class="govuk-link govuk-!-margin-top-2" role="button" download>View and download your receipt</a>
```

Comment:
I don't think its necessary to add another layer of abstraction `ReceiptDownloadController` if you're only able to download from one place (for now). I would probably put the ReceiptDownloadController in the openEApp controller (whatever is used to render this page) - because they're all related

Reply:
Hmm, I was thinking that before but I wanted to make the files more single responsibility if that makes sense. So putting it in openEApp would mean that file is doing multiple things, maybe it could be a service instead of a controller 🤷

Reply:
if it's a service it would go into CB service, which it already is

as far as single responsibility goes, yes, moving it to openEAppController would be correct. From what I can tell the app is using the MVC pattern. really simplistically.. Model would the casebook api, view would be the templates, and controller would broker interactions between the view and model. 

openEAppController wouldn't be single responsibility if it was rendering another page for example, or handling a stream that was completely unrelated.

files don't have to be single responsibility themselves

Reply:
Fair enough I see where you're coming from. I've updated the code accordingly. Let me know your thoguhts.